core/src/test/java/org/sonarsource/sonarlint/core/plugin/PluginLoaderTest.java

```diff
@@ -71,18 +75,19 @@ public void plugin_entry_point_must_be_no_arg_public() {
   @Test
   public void define_classloader() throws Exception {
     File jarFile = temp.newFile();
+    File slf4jAdapter = temp.newFile();
     PluginInfo info = new PluginInfo("foo")
       .setJarFile(jarFile)
       .setMainClass("org.foo.FooPlugin")
       .setMinimalSqVersion(Version.create("5.2"));
 
-    Collection<PluginClassLoaderDef> defs = loader.defineClassloaders(ImmutableMap.of("foo", info));
+    Collection<PluginClassLoaderDef> defs = loader.defineClassloaders(ImmutableMap.of("foo", info), slf4jAdapter);
 
     assertThat(defs).hasSize(1);
     PluginClassLoaderDef def = defs.iterator().next();
     assertThat(def.getBasePluginKey()).isEqualTo("foo");
     assertThat(def.isSelfFirstStrategy()).isFalse();
-    assertThat(def.getFiles()).containsOnly(jarFile);
+    assertThat(def.getFiles()).containsOnly(jarFile, slf4jAdapter);
```

Comment:
The `PluginLoader` adds `slf4jAdapter` as the first entry. Is that important? If yes then perhaps the assertion should be more strict to enforce the ordering.

Reply:
good point, fixed